src/components/Message/MessageSimple/MessageTextContainer.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useContext } from 'react';
+
+import { capitalize } from './utils/capitalize';
+import { renderText } from './utils/renderText';
+
+import { styled, ThemeContext } from '../../../styles/styledComponents';
+
+import type { MessageSimpleProps } from './MessageSimple';
```

Comment:
I'd order this higher up based on it being `./` instead of `../../../`

Reply:
i've been lumping all local import types into one block, ordered by depth and abc, but can talk about newline-ing them too if we want